mops.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "new-base"
-version = "0.1.1"
+version = "0.1.2"
```

```suggestion
version = "0.2.0"
```
since #203 is a breaking change